mule-extensions-api-dsql/src/main/resources/META-INF/mule-module.properties

```diff
@@ -1,4 +1,3 @@
 module.name=mule-extensions-api-dsql
 
-artifact.export.classPackages=org.mule.runtime.extension.api.dsql
```

Comment:
because it's already exported by extensions-api. It's invalid to export a package which is already exported by a dependency